src/stache/includes/v4-note.html

```diff
@@ -0,0 +1,4 @@
+<p></p>
+<sky-alert alertType="warning">
+  This site describes an old version of the SKY UX framework. We still support this version, but we recommend the most recent version. For documentation on the most recent version of SKY UX, see <a href="https://developer.blackbaud.com/skyux">the version 5 website</a>.
```

Comment:
"We still support this version..." This statement won't be true forever, correct? I'd hate to have to come in here and update the v4 docs again after a certain period of time. What if we just omit this sentence entirely?